package io.connorwyatt.flashcards.helpers.auth.exceptions

class AlreadySignedInException : AuthException()
{
}
